proposals/features-library.md

```diff
@@ -72,15 +101,55 @@ function common_function() {
 ```
 
 ```bash
-# featureA/install.sh
+# src/featureA/install.sh
 
 # Include common functions
-source "./utils/common.sh"
+source "../../utils/common.sh"
 
 # Use common function
 common_function "devcontainers"
 ```
 
+
+## [Proposal B] Change: Structure of packaged Feature
+
+To simplify the proposed change above and keep the relative paths consistent before and after packaging, the structure of the packaged Feature will change to mirror the directory structure of the source code.
+
+When running the packaging step, the stage is provided a source directory for packaging >1 Feature, or the root of a project when packaging a single Feature.  The Feature contents will be placed into `./src/<FeatureID>`.  Other included directories will be copied into the root of the packaged artifact as indicated in **Proposal A**.
+
+An implementation should resolve the `devcontainer-feature.json` by checking for its presence in the following order of precedence:
+
+ - `./src/<feature-id>/devcontainer-feature.json`
```

Comment:
I wonder if we need to set `src` in stone. We could copy the `devcontainer-feature.json` to the root folder - probably updating relative paths in its properties, so they do not break. Or: Is it possible to have a symbolic link inside the tar file, making `devcontainer-features.json` at the root a link to the actual location?